src/components/artwork_filter/index.tsx

```diff
@@ -31,9 +32,22 @@ class ArtworkFilter extends React.Component<Props, State> {
       dimension_range: "*",
       price_range: "*",
       medium: "*",
+      loading: false,
     }
   }
 
+  handleLoadMore() {
+    this.setState({ loading: true }, () => {
+      this.props.relay.setVariables({
+        size: this.props.relay.variables.size + PageSize,
+      }, readyState => {
+        if (readyState.done) {
+          this.setState({ loading: false })
+        }
+      })
+    })
```

Comment:
Oh wait, you couldn't get rid of the first `setState` call probably, because I guess you want to trigger a render? 🤔